roles/edpm_libvirt/templates/edpm_libvirt_guests.service

```diff
@@ -10,8 +10,8 @@ Documentation=https://libvirt.org
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/libvirt-guests
-ExecStart=/bin/podman exec libvirt_virtproxyd /bin/rm -f /var/lib/libvirt/libvirt-guests
-ExecStop=/bin/podman exec libvirt_virtproxyd /bin/sh -x /usr/libexec/libvirt-guests.sh shutdown
+ExecStart=/bin/rm -f /var/lib/libvirt/libvirt-guests
```

Comment:
could we use the default unit shipped with the libvirt-daemon/-common packages? We had to maintain out copy because of the need to wrap it with podman. But we no longer need to do this after de-containerization.

Reply:
we also need the addtion fo Before=edpm_nova_compute.service

i could use dropin files to get a similar effect but i fell like that is more likely to break going forward.